airflow/airflow.go

```diff
@@ -21,13 +21,16 @@ const (
 	defaultAirflowVersion = uint64(0x1) //nolint:gomnd
 	componentName         = "airflow"
 
-	airflowVersionLabelName        = "io.astronomer.docker.airflow.version"
 	triggererAllowedAirflowVersion = "2.2.0"
+	triggererAllowedRuntimeVersion = "4.0.0"
```

Comment:
```suggestion
	triggererAllowedRuntimeVersion = "4.2.5"
```

Reply:
I think the triggerer resource was introduced in runtime v 4.0.0

Reply:
it's less important, Gen1 works only starting 4.2.5.

Reply:
https://github.com/astronomer/houston-api/blob/main/src/lib/constants/index.js#L209 i still need to update it here after i merge and release https://github.com/astronomer/astro-runtime/pull/235